fe/fe-core/src/main/java/org/apache/doris/PaloFe.java

```diff
@@ -232,63 +232,67 @@ private static CommandLineOptions parseArgs(String[] args) {
-                        }
+                }
+                String fromKey = "";
+                String endKey = "";
+                int metaVersion = 0;
+                if (cmd.hasOption('f') || cmd.hasOption("from")) {
+                    fromKey = cmd.getOptionValue("from");
+                    if (Strings.isNullOrEmpty(fromKey)) {
+                        System.err.println("from key is missing");
+                        System.exit(-1);
                     }
-                    if (cmd.hasOption('t') || cmd.hasOption("to")) {
-                        endKey = cmd.getOptionValue("to");
-                        if (Strings.isNullOrEmpty(endKey)) {
-                            System.err.println("end key is missing");
-                            System.exit(-1);
-                        }
+                }
+                if (cmd.hasOption('t') || cmd.hasOption("to")) {
+                    endKey = cmd.getOptionValue("to");
+                    if (Strings.isNullOrEmpty(endKey)) {
+                        System.err.println("end key is missing");
+                        System.exit(-1);
                     }
-                    if (cmd.hasOption('m') || cmd.hasOption("metaversion")) {
-                        try {
-                            metaVersion = Integer.valueOf(cmd.getOptionValue("metaversion"));
-                        } catch (NumberFormatException e) {
-                            System.err.println("Invalid meta version format");
-                            System.exit(-1);
-                        }
+                }
+                if (cmd.hasOption('m') || cmd.hasOption("metaversion")) {
+                    try {
+                        metaVersion = Integer.valueOf(cmd.getOptionValue("metaversion"));
+                    } catch (NumberFormatException e) {
+                        System.err.println("Invalid meta version format");
+                        System.exit(-1);
                     }
-
-                    BDBToolOptions bdbOpts = new BDBToolOptions(false, dbName, false, fromKey, endKey, metaVersion);
-                    return new CommandLineOptions(false, "", bdbOpts);
+                }
+                
+                BDBToolOptions bdbOpts = new BDBToolOptions(false, dbName, false, fromKey, endKey, metaVersion);
+                return new CommandLineOptions(false, "", bdbOpts);
                 }
             } else {
                 System.err.println("Invalid options when running bdb je tools");
                 System.exit(-1);
             }
-        } else if (cmd.hasOption('h') || cmd.hasOption("helper")) {
-            String helperNode = cmd.getOptionValue("helper");
-            if (Strings.isNullOrEmpty(helperNode)) {
-                System.err.println("Missing helper node");
-                System.exit(-1);
-            }
-            return new CommandLineOptions(false, helperNode, null);
         }
 
         // helper node is null, means no helper node is specified
```

Comment:
This annotation is confusing. I think it's wrong because the last `return` isn't related with `helper node`

Reply:
The annotation here is to explain the 2nd param of `CommandLineOptions`